docs/deploy/create_server.rst

```diff
@@ -10,39 +10,44 @@ As with other deployment tasks, do the :ref:`setup tasks<generic-setup>` before
 
 Create the server via the :ref:`host<hosting>`'s interface.
 
-Bytemark
-~~~~~~~~
+Linode
+~~~~~~
 
-#. `Login to Bytemark <https://panel.bytemark.co.uk>`__
-#. Click *Servers* and *Add a cloud server*
+#. `Log into Linode <https://login.linode.com/>`__
+#. Click *Create Linode*
 
-   #. Enter a *Name*
-   #. Select a *Group* indicating the environment (*production* or *development*)
-   #. Set *Location* to "York"
-   #. Set *Server Resources*
+   #. Set *Distribution* (Operating System) to the "Ubuntu 20.04 LTS" image
+   #. Set the *Region* to *London UK*
+   #. Set *Linode Plan*
+   #. Set *Linode Label* to the server name
+   #. *Add Tags* indicating the environment (*Production* or *Development*)
+   #. Set *Root Password* to a `strong password <https://www.lastpass.com/password-generator>`__
+   #. Check *Enable Backups*
+   #. Click *Create Linode*
 
-      .. note::
+#. Wait for the server to boot (this will take a few minutes)
 
-         If you drag the slider to the minimum, the disc size will be changed to 5 GiB, which is too little.
+#. Configure Instance disks
```

Comment:
If "Instance disks" is text from the UI, surround in asterisks.

Reply:
"Instance disks" is not from the UI, this item is more of a description of what the following indented steps achieve. 

That said, we don't do this anywhere else in the docs. I've removed this line in a new commit.